src/components/Header/HeaderMobile/HeaderMobileNavigation.tsx

```diff
@@ -23,7 +21,7 @@ export const HeaderMobileNavigation = () => {
     <nav aria-label="Główna nawigacja">
       <ul className="flex flex-col gap-5">
         {orderedNavData.map(({ href, label }, index) => (
-          <React.Fragment key={index}>
+          <Fragment key={index}>
```

Comment:
I believe this component should be refactored and decoupled into smaller parts, even within the same file to make it more readable. We can create an improvement task for this